pkg/clients/aws/client_test.go

```diff
@@ -66,22 +69,67 @@ func TestGetRancherLicense(t *testing.T) {
 			desiredLicense:    rancherProductSKUEmea,
 			errDesired:        false,
 		},
+		{
+			name:                  "test non-emea test license",
+			hasTestNonEmeaLicense: true,
+			usesTestIds:           true,
+			includeProductSku:     true,
+			desiredLicense:        rancherProductTestSKUNonEmea,
+		},
+		{
+			name:               "test emea test license",
+			hasTestEmeaLicense: true,
+			usesTestIds:        true,
+			includeProductSku:  true,
+			desiredLicense:     rancherProductTestSKUEmea,
+		},
+		{
+			name:                  "test non-emea + emea test license",
+			hasTestNonEmeaLicense: true,
+			hasTestEmeaLicense:    true,
+			usesTestIds:           true,
+			includeProductSku:     true,
+			desiredLicense:        rancherProductTestSKUNonEmea,
+		},
+		{
+			name:                  "test non-emea test + prod skus, test requested, test used",
+			hasNonEmeaLicense:     true,
+			hasTestNonEmeaLicense: true,
+			usesTestIds:           true,
+			includeProductSku:     true,
+			desiredLicense:        rancherProductTestSKUNonEmea,
+		},
+		{
+			name:                  "test non-emea test + prod skus, prod requested, prod used",
+			hasNonEmeaLicense:     true,
+			hasTestNonEmeaLicense: true,
+			usesTestIds:           false,
+			includeProductSku:     true,
+			desiredLicense:        rancherProductSKUNonEmea,
+		},
 	}
 	for _, test := range tests {
 		test := test
 		t.Run(test.name, func(t *testing.T) {
```

Comment:
Nano-nits, feel free to completely ignore:
- You could run the subtests in parallel
- Maybe move the `fakeAccountNum` constant inside the test?
- `Use context.TODO` instead of `Background` when you call `GetRancherLicense` in the test (semantically they are slightly different, but functionally the same). `TODO` means you don't care about the context, it's a placeholder.